src/Weather.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import Button from 'react-bootstrap/Button';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import axios from 'axios'
+
+export default class Weather extends Component {
```

Comment:
HI there!

Lab 9 asks that we create new components:

- Within this `Weather` component, create a nested component, `WeatherDay`, to render a single day's forecast. Map over the result to convert the raw response data into an array of these components (see task card).